src/components/ProfileCard/ProfileContent.tsx

```diff
@@ -143,14 +147,26 @@ export const ProfileContent: React.FC<ProfileCardProps> = props => {
             <div className={classes.column}>
               <Typography component="span">{i18n.t('Profile_Card.Network')}</Typography>
               <Typography component="span" className={classes.flex}>
-                {getSelectedIcon()} {formatNetworkTitle(currentWallet?.network)}
+                {isWeb2Users ? (
+                  'Not Connected'
+                ) : (
+                  <>
+                    {getSelectedIcon()} {formatNetworkTitle(currentWallet?.network)}
+                  </>
+                )}
               </Typography>
             </div>
             <div className={classes.column}>
               <Typography component="span">{i18n.t('Profile_Card.Wallet')}</Typography>
               <Typography component="span" className={classes.flex}>
-                {getSelectedIcon(true)}
-                {formatWalletTitle(currentWallet?.network)}
+                {isWeb2Users ? (
+                  'Not Connected'
```

Comment:
And this too

Reply:
resolved in https://github.com/myriadsocial/myriad-web/pull/1542/commits/5388d14ad4af4a00aac4e930004ea3a7ca9981e9 @irmannmal